packages/odyssey-react/src/components/Box/Box.tsx

```diff
@@ -17,10 +17,12 @@ import {
   withStyles,
   PolymorphicForwardRef,
   toPascalCase,
+  toCamelCase,
 } from "../../utils";
 import styles from "./Box.module.scss";
+import textStyles from "../Text/Text.module.scss";
```

Comment:
dang - wish we didn't have to use this approach but I understand why you did here. This is brittle because if we refactor in Text we can inadvertently break Box now, as there is no type safety for CSS modules.

Its also particularly confusing because Text composes Box internal to itself and not the other way around so its somewhat strange for Box to consumes Text's styles (its parent).

Lastly, I'm pretty sure that any values imported from Text like this won't be themeable even if they are defined as such for Text. This is because although they will be transpiled by our build tooling they won't be exposed to the HOC below in the same way `import styles from "./Box.module.scss";` is.

Reply:
bumping this comment @andrewberg-okta do you have any thoughts to share here?

Reply:
In working through this in order for those imported styles to work they have to be exposed to the HOC in the same way. https://github.com/okta/odyssey/pull/1201/files#diff-27aca65767e8a0cb5a820f200969dfea1e8a356048de0f7465380be3afe92f98R565

But I'm not sure of the full implications of doing it that way

Reply:
Regardless it's better not to have the tight coupling, moving the styles into Box